_posts/en/newsletters/2022-05-04-newsletter.md

```diff
@@ -0,0 +1,188 @@
+        calling function can use whichever one is appropriate for its
+        context.
+
+    - *Interaction with scriptpath spends:* keys meant to be used with
+      scriptpath spends have a related problem: in order to use the
+      scriptpath, the spender must know the pre-tweaked key.  Again,
+      this suggests that implementations return the pre-tweaked key so
+      that it's available to be used in other code that needs it.
+
+    - *Shortcut for final signer:* Osuntokun also sought clarification
+      on a section in the BIP which describes how the final signer (and
+      only the final signer) can use deterministic randomness or a
+      lower-quality source of randomness for generating their signature
+      nonce.  Brandon Black [replied]black musig2] to describe the
+      situation that had motivated the section---they had a signer that
+      would have a difficult time securely managing a regular MuSig2
+      signing session but which they were instead able to always use as
+      the final signer.
+
+- **Measuring user support for consensus changes:** Keagan McClelland
+  [posted][mcclelland measure] to the Bitcoin-Dev mailing list a
+  proposal similar to [previous proposals][bishop signal] to have
+  transactions signal whether or not they [supported][topic soft fork
+  activation] a particular proposal to change the consensus rules.  In
+  the thread, several related sentiment measurement ideas were also
+  discussed, but all appeared to have problems, such as [technical
+  challenges][aronesty signal parse scripts], significantly
+  [reducing][grant signal chainalysis] user privacy, [favoring][tetrud
+  signal favor] certain parts of the Bitcoin economy over others, or
+  [penalizing][ivgi signal hodl voting] early voters over those who
+  waited to participate in consensus formation.
+
+    As on previous occasions where this topic has been discussed, it
+    did not appear that any of the suggested methods would produce a
+    result that would be sufficiently respected by most of the
+    discussion participants to strongly inform their decisions about
+    changing Bitcoin's consensus rules.
+
+- **LN anchor outputs security issue:** Bastien Teinturier
+  [posted][teinturier security] to the Lightning-Dev mailing list the
+  announcement of a security issue he had previously responsibly
+  disclosed to LN implementation maintainers.  The issue affected older
+  versions of Core Lightning (with experimental features enabled) and
+  LND.
+
+    Prior to the implementation of [anchor outputs][topic anchor
+    outputs], revoked [HTLC][topic HTLC] transactions only contained a
+    single output, so many implementations only tried to claim that
+    single output.  The design of anchor outputs for LN allows combining
+    multiple revoked HTLC outputs into a single transaction, but this
+    is only safe if implementations claim all of the relevant outputs in
```

Comment:
If I understand this right, the issue was that some implementations would not claim all funds that belonged to them. If so, I think the "is only safe" put me on the wrong track here. Perhaps this could be rephrased to clarify—especially if I'm still on the wrong track. :)

Reply:
Updating to make it clearer that it's a theft issue, thanks for letting me know about the ambiguity!